content/en/docs/score specification/score-spec-reference.md

```diff
@@ -30,27 +30,27 @@ apiVersion: string
 
 metadata:
   name: string
+  annotations: # optional
+    annotations-name: string # optional
 ```
 
-`apiVersion`: the declared Score Specification version.
+`apiVersion`: the declared Score Specification version. Find the current version [here](https://github.com/score-spec/spec/blob/main/score-spec-v1b1.yaml).
```

Comment:
Shouldn't we link to the real spec? https://github.com/score-spec/schema/blob/main/score-v1b1.json

Reply:
Done ✅